bfv/marshaler.go

```diff
@@ -1,66 +1,21 @@
 package bfv
```

Comment:
Since marshalers are well modularized now, I think there is no point in keeping `bfv/marshaler.go`.  The methods could be moved to `bfv/ciphertext.go`.

Reply:
fixed